chainer/link.py

```diff
@@ -554,13 +554,11 @@ def cleargrads(self):
     def zerograds(self):
         """Initializes all gradient arrays by zero.
 
-        This method can be used for the same purpose of cleargrads, but less
-        efficient. This method is left for backward compatibility.
+         .. deprecated:: v1.15
+            Use :meth:`cleargrads` instead.
 
-        .. deprecated:: v1.15
-           Use :meth:`cleargrads` instead.
+         """
```

Comment:
* Indentation seems broken.
* How about saying `Use :meth:`cleargrads` instead, which is more efficient.`? So users can understand this is not just a rename.

Reply:
Updated.